packages/compartment-mapper/test/test-policy.js

```diff
@@ -100,53 +109,60 @@ const anyExpectations = {
   },
 };
 
-const fixtureAssertionCount = 2;
-const makeAssertions =
+const makeResultAssertions =
   expectations =>
-  async (t, { namespace, compartments }) => {
+  async (t, { namespace }) => {
     t.deepEqual(namespace, expectations.namespace);
-
-    await t.throwsAsync(
-      () => compartments.find(c => c.name.includes('alice')).import('hackity'),
-      { message: /Failed to load module "hackity" in package .*alice/ },
-      'Attempting to import a package into a compartment despite policy should fail.',
-    );
   };
 
+const assertNoPolicyBypassImport = async (t, { compartments }) => {
+  await t.throwsAsync(
+    () => compartments.find(c => c.name.includes('alice')).import('hackity'),
+    { message: /Failed to load module "hackity" in package .*alice/ },
+    'Attempting to import a package into a compartment despite polict should fail.',
```

Comment:
```suggestion
    'Attempting to import a package into a compartment despite policy should fail.',
```

Reply:
The number of times I already fixed this typo...  :sweat_smile: